src/main/scala/xiangshan/cache/dcache/mainpipe/MainPipe.scala

```diff
@@ -1418,11 +1418,11 @@ class MainPipe(implicit p: Parameters) extends DCacheModule with HasPerfEvents w
 
   req.ready := s0_can_go
 
-  io.meta_read.valid := req.valid && s1_ready && !set_conflict
+  io.meta_read.valid := req.valid && !set_conflict
   io.meta_read.bits.idx := get_idx(s0_req.vaddr)
   io.meta_read.bits.way_en := Mux(s0_req.replace, s0_req.replace_way_en, ~0.U(nWays.W))
 
-  io.tag_read.valid := req.valid && s1_ready && !set_conflict && !s0_req.replace
+  io.tag_read.valid := req.valid && !set_conflict && !s0_req.replace
```

Comment:
If `MainPipe` gets blocked for some reason and `s1_ready` stays deasserted, `MainPipe` will occupy the read ports of meta array and tag array. Won't this impair the performance?

Reply:
metaArray is a register group, so load and mainpipe reading do not affect each other. 
tagArray is a duplicate SRAM. Each reading port has its own copy of SRAM. Load and mainpipe reading will not affect each other.
So it might not affect performance here.